### ISSUE TITLE
Splitting reflection into three tabs

### ISSUE THREAD

**Issue by dantaeyoung** [lmccart/the-changing-room#95]
Hi @dantaeyoung following up on this area after the tech test.separate browser tabs, each fullscreen and sent to projector output. The three different tabs could have address:
```
http://localhost:3000/02-reflection?screen=0
http://localhost:3000/02-reflection?screen=1
http://localhost:3000/02-reflection?screen=2
```

The design would stay mostly the same (with the white borders and image reflection as planned. However, the text should go only on the middle screen, and the popups should be positioned so they're fully within their own tab view. There will be some slight async with the different tabs, but I think it's preferable and easier than mirroring all three sides the same somehow.

_Originally posted by @lmccart in https://github.com/lmccart/the-changing-room/issues/31#issuecomment-761874086_

**Comment by dantaeyoung** [comment]
Summary
    screen 0: memory image popups only
    screen 1: meditation text & memory text popups
    screen 2: memory image popups only

with loading screen visible on all three tabs

**Comment by dantaeyoung** [comment]
Closed by #100
